libraries/AP_HAL_ChibiOS/hwdef/CUAV-Nora/hwdef.dat

```diff
@@ -0,0 +1,311 @@
+# hw definition file for processing by chibios_hwdef.py
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# crystal frequency
+OSCILLATOR_HZ 16000000
+
+# USB setup
+USB_VENDOR 0x0483 # ST
+USB_PRODUCT 0x5740
+USB_STRING_MANUFACTURER "ArduPilot"
+USB_STRING_PRODUCT "%BOARD%"
+USB_STRING_SERIAL  "%SERIAL%"
+
+
+# board ID for firmware load
+APJ_BOARD_ID 1009
+
+FLASH_SIZE_KB 2048
+
+# with 2M flash we can afford to optimize for speed
+env OPTIMIZE -O2
+
+# ChibiOS system timer
+STM32_ST_USE_TIMER 2
+
+# bootloader takes first sector
+FLASH_RESERVE_START_KB 128
+
+# board voltage
+STM32_VDD 330U
+
+# order of UARTs (and USB)
+UART_ORDER OTG1 USART1 USART2 USART6 UART4 UART8 UART7 OTG2
+
+# default the 2nd interface to MAVLink2 until MissionPlanner updates drivers
+define HAL_OTG2_PROTOCOL SerialProtocol_MAVLink2
+
+define CONFIG_HAL_BOARD_SUBTYPE HAL_BOARD_SUBTYPE_CHIBIOS_FMUV5
```

Comment:
I don't think we need or want these FMUv5 defines. This was fixed in master today

Reply:
Do I need to delete this definition?